packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1288,7 +1289,7 @@ function updateEffect(
     }
   }
   return updateEffectImpl(
-    UpdateEffect | PassiveEffect,
+    UpdateEffect | PassiveEffect | PassiveStaticEffect,
```

Comment:
Don't think you need it here, only on mount. Since it's not removed during clone.

Reply:
Yeah I guess that's fair.